.github/workflows/build.yml

```diff
@@ -0,0 +1,45 @@
+name: Check
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
+
+env:
+  CARGO_TERM_COLOR: always
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Install toolchain
+        uses: dtolnay/rust-toolchain@stable
```

Comment:
```suggestion
        uses: dtolnay/rust-toolchain@nightly
```

Reply:
https://github.com/open-web3-stack/XCQ/actions/runs/9076865469 This is action log

Reply:
I was thinking why rust-toolchain@stable is invalid.

Reply:
`
dtolnay/rust-toolchain@stable is not allowed to be used in open-web3-stack/XCQ. Actions in this workflow must be: within a repository owned by open-web3-stack, created by GitHub, verified in the GitHub Marketplace, or matching the following: actions-rs/*, actions/*, chevdor/srtool-actions@*, codecov/codecov-action@*, dtolnay/rust-toolchain@nightly, actions-rust-lang/setup-rust-toolchain@*, swatinem/rust-cache@*, taiki-e/install-action@v2, chainguard-dev/actions/setup-gitsign@main, cargo-bins/release-pr@v2.
`

We limit the Actions. If you want to add a new one, you need to pin the owner(Bryan)